src/renderer/renderer.ts

```diff
@@ -2453,7 +2498,7 @@ export class SDFGRenderer {
         if (this.external_mouse_handler) {
             const ext_mh_dirty = this.external_mouse_handler(
                 evtype, event, { x: mouse_x, y: mouse_y }, elements,
-                this, this.selected_elements, ends_drag, this.sdfv_instance
+                this, this.selected_elements, this.sdfv_instance
```

Comment:
What's this sneaky line doing here?

Reply:
This is in preparation for being able to create groupings from VSCode via box select. When multiple elements are selected via Ctrl.+Click, the information panel stops showing details from a specific element and instead displays a message saying 'multiple elements selected'. However, this `ends_drag` check was blocking this from happening, if the selection was performed via box select - which makes no sense. I am not sure why this check was there to begin with, but testing didn't bring up any issues with respect to dragging and releasing that would hint to this check being required..

Reply:
Nice!